src/docs/user/references/Variables.adoc

```diff
@@ -286,6 +286,13 @@ _Some characters of the regular expression must be encoded according to the link
 
 _Some characters of the regular expression must be encoded according to the link:https://en.wikipedia.org/wiki/URL_encoding[specification]._
 |?name=token.* => PA:LIST(,token1,token2)
+
+|Tokens Filtered by Node Source
+|All tokens registered in the provided Node Source. This filter will return a list which contains tokens defined globally on the Node Source and on any Node which belongs to the Node Source. See <<_node_access_token>>.
```

Comment:
This filter returns a list containing tokens defined globally in the Node Source and in any node belonging to the Node Source.